ui/app/AppLayouts/Wallet/stores/CurrenciesStore.qml

```diff
@@ -922,8 +922,7 @@ QtObject {
         var isSelected = false
         for(var i = 0; i < currenciesModel.count; i++) {
             if(root.currentCurrency === root.currenciesModel.get(i).key) {
-                isSelected = true
-                root.currenciesModel.get(i).selected = true
+                root.currenciesModel.get(i).selected = isSelected = true
```

Comment:
As mentioned in #6092 this right here will not actually re-render the view with the newly selected currency.